client/components/charts/ScatterChart.tsx

```
@@ -70,6 +71,69 @@ export function ScatterChart({
+  };
+
+  const onUpdate = (_event: any) => {
+    // Plotly単体で設定できないデザインを、onUpdateのタイミングでSVGをオーバーライドして解決する
+
+    // アノテーションの角を丸にする
+    const bgRound = 4
+    document.querySelectorAll('g.annotation').forEach((g) => {
+      const bg = g.querySelector('rect.bg');
+      if (bg) {
+        bg.setAttribute('rx', `${bgRound}px`);
+        bg.setAttribute('ry', `${bgRound}px`);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)